Source/Basic-Conditions-And-Behaviors/Runtime/Conditions/PositionalConditions/ObjectInTargetCondition.cs

```diff
@@ -19,13 +19,16 @@ protected ObjectInTargetActiveProcess(TData data) : base(data)
         public override void Start()
         {
             Data.IsCompleted = false;
+            CheckPossibleErrors();
             isInside = IsInside();
 
             if (isInside)
             {
                 timeStarted = Time.time;
             }
         }
+        
+        protected abstract void CheckPossibleErrors();
```

Comment:
Does it make sense to put this here if you're using it only in your condition, and are adding an empty implementation to the other 2?